account_invoice_refund_link/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2004-2011 Pexego Sistemas Informáticos. (http://pexego.es)
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# Copyright 2014-2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Link refund invoice with original",
```

Comment:
This text must change to `Show links between refunds and their originator invoices`

Reply:
done

Reply:
I prefer that as a `summary`, and the `name` be the proper name of the module hahaha

Reply:
The name is the short description, but this one can't be shorter

Reply:
And 'summary' is the _long_ description? 🤔

Reply:
It's a bit longer, yes.